### ISSUE TITLE
[UX] Pass off Product Guides to the Contact Center

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#38887]
## Description
The product guides need to be passed off to the contact center. Here is the process that we need to follow to do that. We need to set aside time for someone to do this work (doesn't necessarily have to be UX).

## Acceptance Criteria
- [ ]

**Comment by benbrasso-agile6** [comment]
[Pre check in word](https://app.zenhub.com/files/133843125/cfdd0291-b857-482a-b478-2c5f6bde7e5b/download)
[Pre check in pdf](https://app.zenhub.com/files/133843125/5dbcb2a9-a951-4f4d-ba9d-79e6731bf6d5/download)

NOT updated in Git yet. Waiting to finish Check In. I'll do that on Monday. 

Other notes: 

@zach-park in the user flow for pre-check-in, we should look to update the confirmation page, so it includes a link to VA profile. Tag me to discuss or when complete so I can update the product guide. 

Redid the screenshots, but we need to take them from a phone a different time. I can grab a developer to help when I have some time. 

Did we move away from the usa.gov domain? Can that be updated in the product guide?

**Comment by benbrasso-agile6** [comment]
Product guides will need to be updated with DOB changes. Will evaluate when do that based on timing of wanting to send these to Contact Center.

**Comment by benbrasso-agile6** [comment]
Google doc is out of date at this point. Using Word.

**Comment by zach-park** [comment]
Hi @benbrasso-agile6 , here is the updated screenshot for the ID verification page with DOB changes. Thank you!
![Verify Identity](https://user-images.githubusercontent.com/91493630/165115760-db546dc6-809d-4158-b221-2a7311e8bab7.jpg)

**Comment by benbrasso-agile6** [comment]
[Check in Word](https://app.zenhub.com/files/133843125/c430e57e-b139-42d7-be43-82f0695056ee/download)
[Check in PDF](https://app.zenhub.com/files/133843125/4c6b558a-2555-4f9b-b691-1f5e0f2016a0/download)

[Pre check in Word](https://app.zenhub.com/files/133843125/5ebf850e-236c-49dd-b054-7b303b5406f2/download)
[Pre check in PDF](https://app.zenhub.com/files/133843125/1701620e-090b-4734-9936-15ccf83250e8/download)

Notes: 

1) Product guides are updated above. 
2) Added details about updates in version notes in each document. 
3) Would recommend someone do proof reading and/or UX review, especially check in since I'm least familiar with that process. E.g., Kristen and/or Nira. 
4) Documents are no longer updated in Google Drive. Certainly, Word can be converted into Google Drive. 
5) I don't believe there is a "non edit edit screen" for check in. I think it's only pre-check-in that that's available. For example, there's no wireframe to my knowledge for "visit your VA profile to update your information" for check in. 
6) This does NOT include the DOB change. I wasn't sure how the timing of submitting these product guides will compare to when we're releasing that update to prod. So, @loripusey could make a call on whether we should submit these with or without DOB details. 
7) If/when time permits, I might recommend doing an audit of the wireframes compared to actual product and update Sketch accordingly. This matters because the screenshots I included were from Sketch, since those are more easily accessible. Here are some things I noticed that could be updated in Sketch: different VA.gov header, missing some required text, e.g., SSN, different UI elements when comparing pre check in to check in, e.g., address confirmation screen, missing emergency contact screen in Sketch, etc. I don't think this needs to hold up submission to the product guides, per se, but just something to be aware of. 
8) If/when time permits, take time with engineers and UX to discuss how to do screenshots for product guides moving forward. Ideally, they could be screenshots from product vs. Sketch pngs. And then, overlay those screenshots on top of a mobile image in Sketch. Would require some collaboration and document the process for future screenshots. 
9) User flow diagram in pre check in product guide was NOT updated. E.g., it could be updated to include the new "update your VA profile" screen on the confirmation page. Again, not something that I think needs to hold up this process. 
10) Lastly, that's all I was able to get to. There are some remaining tasks left in this ticket, FYI. I'm happy to pick up when I return, as needed. @loripusey @kristenmcconnell

**Comment by benbrasso-agile6** [comment]
Again, I think the idea was to get SOMETHING to contact center soon vs. waiting for DOB, language translations, etc. Of course, will leave that up to product to decide. But, wanted to document this note.

**Comment by loripusey** [comment]
Yes, I think we should give the Contact Center what we have. We will need to repeat the process whenever we update our artifacts, which may be often.

**Comment by loripusey** [comment]
I think that we need to get UX up-to-speed on using Codespaces to view the product in the staging environment. We should have the ability to run the applications and take our own screen shots, I don't think that we should be getting screen shots from the Sketch diagrams. I'll set up some time to get us all using Codespaces.

**Comment by loripusey** [comment]
@benbrasso-agile6 While you were in STL, Nira/Kristen identified some changes that affected this user story; there is a new Sketch diagram for this [story](https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/39273); you may need to revisit changes to the Product Guide et. at. 
@kristenmcconnell @niradatta1

**Comment by benbrasso-agile6** [comment]
@loripusey any idea when that update will be in production? I'm trying to determine when we want to be submitting the product guide to the Contact Center and the training team. Or, did you envision we move forward without these being pushed to production environment yet? 

Reminder: recent features that aren't yet included in the product guide are translations and DOB change. We could add them, but wanted to note that they aren't currently.

**Comment by loripusey** [comment]
The ticket for the work was pulled into this sprint but did not get completed. It will need to get rolled over to the next sprint. I suggest we submit the current documentation to the Contact Center. We will need to go through that process every time we update the documents I'm assuming.

**Comment by benbrasso-agile6** [comment]
Adding materials that Katonya provided to us as reference

[PCI Training Guide for CI and VSE CS](https://app.zenhub.com/files/133843125/bd649f74-313a-4048-9e6f-814750582e0a/download)

[PCI Training Guide with "Hard Stops"](https://app.zenhub.com/files/133843125/f95adba1-ea6e-4882-9e9e-58e14469a160/download)

**Comment by benbrasso-agile6** [comment]
@kristenmcconnell let's catch up on this ticket at some point, i.e., to see if you had any feedback. Otherwise, I will go ahead and publish to Git, contact the Contact Center and push materials to training folks.

**Comment by benbrasso-agile6** [comment]
We'll need to create a Product Video that gets published to the VA YouTube channel and used by the Contact Center. Discussing next steps in slack with CIE team.

**Comment by loripusey** [comment]
Let's schedule a demo with them and record the demo.

**Comment by kristenmcconnell** [comment]
@benbrasso-agile6 Would it be a lot of work to replace the "non edit edit" image within the pre-check-in guide? The version in the current doc will never be implemented. If it is being worked on next sprint, should we make that small change? I sent you the new designs via Slack. We can review this more tomorrow morning when we meet; otherwise, it looks good.

**Comment by benbrasso-agile6** [comment]
Contact Center Support doesn't have to be a part of the Zoom call. We should record it on our own. That will be the most efficient. I started a thread in #vsp-contact-center-support if we have questions. 

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/request-contact-center-review.md

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/product-demo-video-process.md

https://github.com/department-of-veterans-affairs/va.gov-team/blob/69833737d9fe22b8990bb987e7c50de13205c5d5/platform/product-management/demo-video-creation-process.md

@loripusey we'll want to assign some team members to produce the demo video and/or provide scripting support. I'm happy to help, but I think we might need something like: 

1) Engineer to steer the experience, and 

1) Engineer and/or ux/product to do the voiceover

**Comment by benbrasso-agile6** [comment]
Updated non edit edit per Kristen comment
